addons/hr_expense/tests/test_expenses.py

```diff
@@ -118,11 +120,11 @@ def test_expense_main_flow(self):
         # Generate a payment for 'company_account' (and its move(s)) and a vendor bill for 'own_account'
         expense_sheets.action_sheet_move_create()
         self.assertRecordValues(expense_sheets, [
-            {'state': 'post', 'payment_state': 'not_paid', 'accounting_date': date(2021, 10, 10)},
-            {'state': 'done', 'payment_state': 'paid',     'accounting_date': date(2021, 10, 12)},  # Set to paid as move is posted directly
+            {'state': 'post', 'payment_state': 'not_paid',    'accounting_date': date(2021, 10, 10)},
+            {'state': 'done', 'payment_state': 'paid',        'accounting_date': False},  # Set to paid as move is posted directly
```

Comment:
I still don't understand why the `account_date` of this expense sheet isn't set while the other date are.
Especially with the `state` being `done` and the `payment_state` being `paid`.